manager/integration/negative_tests/negative.robot

```diff
@@ -0,0 +1,39 @@
+*** Settings ***
+Documentation    Negative Test Cases
+
+Library    ../libs/common.py
+
+*** Test Cases ***
+Replica Rebuilding With Replica Deletion
+    Create a volume 5 GB with 3 replicas
+    Write data to the volume
+    FOR    ${i}    IN RANGE    200
+        Delete replica 0 to trigger replica rebuilding
```

Comment:
`Delete replica 0 to trigger replica 0 rebuild`

Reply:
Anyone can write a new test case skeleton. Should we need a rule about how to describe a test step?

Reply:
Sounds good to me. Initially, this can be based on review and then figuring out the pattern we would like.